services/geocoding.py

```diff
@@ -45,7 +47,7 @@ def enhance(self, entries: List[PointOfInterest]) -> List[PointOfInterest]:
         coordinates: Mapping[str, Point] = {}
 
         for address in addresses_to_geocode:
-            geoData = make_geocode_request(address, gmaps)[0]["geometry"]["location"]  
+            geoData = self.make_geocode_request(address, gmaps)[0]["geometry"]["location"]  
```

Comment:
Actually, I'm not sure it's ok to convert to `float` that violates the schema of the `PointOfInterest` model...